model/src/main/kotlin/AnalyzerRun.kt

```diff
@@ -21,10 +21,22 @@ package com.here.ort.model
 
 import com.here.ort.model.config.AnalyzerConfiguration
 
+import java.time.Instant
+
 /**
  * To summary of a single run of the analyzer.
  */
 data class AnalyzerRun(
+        /**
+         * The [Instant] the analyzer was started. The default value exists only for backward compatibility.
+         */
+        val startTime: Instant = Instant.EPOCH,
```

Comment:
Would it make sense to use either a long or a string to eliminate the dependency on Instant.toString ?

Reply:
It's not using `toString()`, see above.